openy_prgf/modules/openy_prgf_simple_content/SMOKE_TESTS.md

```diff
@@ -0,0 +1,72 @@
+# Welcome to Simple Content smoke tests documentation
+
+In order for Open Y Simple Content being tested in a short timeframe, please follow steps below
+
+# Paragraph: Simple Content								
+
+## Adding paragraph to a page
+
+### User
+
+Administrator
+
+### Steps
+
+1. Login as Admin 
+2. Go to Content page -> Add Content
+3. Create a new landing page
+4. In the Content area verify you can add a paragraph called ""Simple content""
+5. Verify paragraph has only one field called ""Content"" and it is required
+6. Verify the field has text editor enabled
+7. Verfiy there are no errors and page created successfully using paragraph
+8. Verify you can edit the page, edit paragraph, without any issues
```

Comment:
Do we need to add a step to check content embedding into simple content (for example, image, video)?

Reply:
Anyway, this can be part of editorial component documentation.